build.sbt

```diff
@@ -5,20 +5,20 @@ import uk.gov.hmrc.DefaultBuildSettings.addTestReportOption
 
 val appName = "accessibility-statement-frontend"
 
-val silencerVersion = "1.7.0"
+val silencerVersion = "1.7.9"
```

Comment:
I think the silencer plugin can be removed completely, as there's native support for suppressing warnings now?